core/src/test/scala/unit/kafka/server/ReplicationQuotasTest.scala

```diff
@@ -86,7 +86,7 @@ class ReplicationQuotasTest extends ZooKeeperTestHarness {
     brokers = (100 to 105).map { id => TestUtils.createServer(fromProps(createBrokerConfig(id, zkConnect))) }
 
     //Given six partitions, lead on nodes 0,1,2,3,4,5 but will followers on node 6,7 (not started yet)
```

Comment:
Did you mean to say `leader` instead of `lead`? Also, the bit after `but` looks like it could be clarified.